composer.json

```diff
@@ -11,11 +11,11 @@
         }
     ],
     "require": {
-        "php": ">=7.1",
-        "dragonbe/vies": "^2.1",
-        "symfony/http-kernel": "^3.0||^4.0",
-        "symfony/dependency-injection": "^3.0||^4.0",
-        "symfony/validator": "^3.0||^4.0"
+      "php": ">=7.1",
+      "dragonbe/vies": "^2.1",
+      "symfony/http-kernel": "^3.0||^4.0",
+      "symfony/dependency-injection": "^3.0||^4.1.12",
```

Comment:
This still leaves open `>= 3.0.0, < 3.4.26` & `>= 4.2.0, < 4.2.7`. While you're at it, can we add `^5.0`?

Reply:
I will add the SF5 changes in a seperate PR